src/Silverback.Integration.Kafka/Messaging/Broker/KafkaConsumer.cs

```diff
@@ -264,7 +262,12 @@ protected override ISequenceStore GetSequenceStore(KafkaOffset brokerMessageIden
         /// <inheritdoc cref="Consumer{TBroker,TEndpoint,TIdentifier}.CommitCoreAsync(IReadOnlyCollection{IBrokerMessageIdentifier})" />
         protected override Task CommitCoreAsync(IReadOnlyCollection<KafkaOffset> brokerMessageIdentifiers)
         {
-            var lastOffsets = brokerMessageIdentifiers
+            if (!Endpoint.Configuration.IsGroupIdSet)
+            {
```

Comment:
I don't enforce adding brackets in the if statements and I usually omit them for single lines (when readable enough).
It would be better if you could stick to the in-use code style, but it's really just a detail.

Reply:
Fixed, see: https://github.com/BEagle1984/silverback/pull/172/commits/62e9b8b415772dfaab2ac3f648be3fb709e33cde